std_msgs/Header header

float64 h_meas_x
float64 h_meas_y
float64 h_meas_x_NED
float64 h_meas_y_NED
float64 h_mahalanobis
int8 h_outlier_rejected
float64[2] h_innovation_vec
float64[4] h_innovation_mat
float64[8] h_k

float64 v_meas_z
float64 v_meas_z_NED
float64 v_mahalanobis
int8 v_outlier_rejected
float64 v_innovation_vec
float64 v_innovation_mat
float64 v_k

float64 r_meas_yaw
float64 r_meas_yaw_NED
float64 r_mahalanobis
int8 r_outlier_rejected
float64 r_innovation_vec
float64 r_innovation_mat
float64 r_k

